otpl/pddl/test_domains/temporal_logistics_uncertainties.json

```diff
@@ -0,0 +1,13 @@
+{"actions": [
+    {"name": "walk", "uncertainty": 0.1},
+    {"name": "drive_truck", "uncertainty": 0.2},
+    {"name": "move_plane", "uncertainty": 0.05},
+    {"name": "move_boat", "uncertainty": 0.05}
+],
+"tils": [
+    {"name": "(not (deliverable pack1))", "uncertainty": 0.1},
+    {"name": "(not (deliverable pack2))", "uncertainty": 0.05},
+    {"name": "(not (deliverable pack3))", "uncertainty": 0.05},
+    {"name": "(not (deliverable pack4))", "uncertainty": 0.05}
+]
+}
```

Comment:
This is not relevant to PIP refactoring, it should be removed from the PR.

Reply:
If the other temporal planning files are to be included (e.g. making this a PIP and PSTN PR) that's OK too.